sql/acceptance/sql_dashboard_test.go

```diff
@@ -57,6 +57,7 @@ func TestAccDashboard(t *testing.T) {
 					name = "My Table"
 
 					options = jsonencode({})
+					query_plan = jsonencode({})
```

Comment:
If we omit it, then will it always cause the problems with visualizations?  If yes, can we provide sensible defaults?

Reply:
can we add `tf:"suppress_diff"` to entity schema?

Reply:
It depends on the data in `options` whether or not it is needed. There is a `version` field in options to indicate.

Reply:
What do we need to suppress?

I modified the integration test to confirm that the value propagates and is consistent after apply.

Reply:
We need to document that we need `query_plan` if we have `version` in parameters. I may need to modify exporter to handle it as well